src/controllers/customercontroller.js

```diff
@@ -6,29 +6,81 @@ class CustomerController extends OrderController
             "src/views/template/customer.html",
             "#customers",
             {}
-        );
+        ).done(() => this.registerEvent());
+    }
+
+    registerEvent()
+    {
+        $(document).on('click', '.customer-complete', function(e) {
+            e.preventDefault();
+
+            var id = e.target.parentNode.parentNode.id.split('-')[1];
```

Comment:
I think `$(this).closest("div:data(customer)").data("customer")` should work also, when using data attributes.